devtools/BUILD.bazel

```diff
@@ -0,0 +1,43 @@
+load("@bazel_skylib//lib:selects.bzl", "selects")
+
+selects.config_setting_group(
```

Comment:
We have similar config setting groups defined in `rust-deps/BUILD.bazel` for the cargo_bazel binary. We should define these in the build folder so that we only need one group of them. For completion, we could also add darwin-amd64 since we have it there (or not, if we assume that no one working on this code base is on Intel Mac).